integration/shared/isolated/add_network_policy_command_test.go

```diff
@@ -84,6 +86,42 @@ var _ = Describe("add-network-policy command", func() {
 				Eventually(session).Should(Exit(0))
 			})
 
+			When("destination space and org are specified", func() {
+				var (
+					sourceOrg   string
+					sourceSpace string
+					sourceApp   string
+				)
+
+				BeforeEach(func() {
+					sourceOrg = helpers.NewOrgName()
+					sourceSpace = helpers.NewSpaceName()
+					sourceApp = helpers.PrefixedRandomName("sourceapp")
+
+					helpers.SetupCF(sourceOrg, sourceSpace)
+
+					helpers.WithHelloWorldApp(func(appDir string) {
+						Eventually(helpers.CF("push", sourceApp, "-p", appDir, "-b", "staticfile_buildpack", "--no-start")).Should(Exit(0))
+					})
+				})
+
+				It("creates a policy", func() {
+					session := helpers.CF("add-network-policy", sourceApp, "--destination-app", appName, "-o", orgName, "-s", spaceName)
+					username, _ := helpers.GetCredentials()
+					Eventually(session).Should(Say(`Adding network policy to app %s in org %s / space %s as %s\.\.\.`, sourceApp, sourceOrg, sourceSpace, username))
+					Eventually(session).Should(Say("OK"))
+					Eventually(session).Should(Exit(0))
+
+					// TODO use helpers.CF("network-policies") after that's updated
```

Comment:
Could this be replaced with `cf curl` until needed functionality are added to the `network-policies` command. We didnt see any upcoming stories for `cf network-policies`.

Reply:
Looks like there is still some design decisions our PM wants to make before working on `cf network-policies`. Replaced with a curl to verify the policy was added.

Reply:
Hi @Ema93sh please let us know if we are a blocker for merging this pull request as I'm just seeing these comments at EOD. Thank you.